adam-core/src/main/scala/org/bdgenomics/adam/converters/AlignmentRecordConverter.scala

```diff
@@ -155,8 +154,8 @@ class AlignmentRecordConverter extends Serializable {
         // only set alignment flags if read is aligned
         if (m) {
           // if we are aligned, we must have a reference
-          assert(adamRecord.getContig != null, "Cannot have null contig if aligned.")
-          builder.setReferenceName(adamRecord.getContig.getContigName)
+          assert(adamRecord.getContigName != null, "Cannot have null contig if aligned.")
```

Comment:
Small nit: while we're changing this line, we should probably `s/assert/require/g`.